packages/arb-token-bridge-ui/src/hooks/__tests__/useBalance.test.tsx

```diff
@@ -16,18 +16,28 @@ const Container = ({ children }: PropsWithChildren<unknown>) => (
   <SWRConfig value={{ provider: () => new Map() }}>{children}</SWRConfig>
 )
 
+const provider = new StaticJsonRpcProvider(
+  process.env.NEXT_PUBLIC_ETHEREUM_RPC_URL,
+  1
+)
+jest.mock('../../token-bridge-sdk/utils', () => ({
+  getProviderForChainId: function A() {
```

Comment:
100% yes, it's a leftover from debug (I would probably rename it to `getProviderForChainIdMock` rather than use anonymous function, it's easier to debug